includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -338,7 +338,7 @@ public function payment_icons() {
 			'wc_stripe_payment_icons',
 			[
 				WC_Stripe_Payment_Methods::ALIPAY      => '<img src="' . WC_STRIPE_PLUGIN_URL . '/assets/images/alipay.svg" class="stripe-alipay-icon stripe-icon" alt="Alipay" />',
-				'wechat'                               => '<img src="' . WC_STRIPE_PLUGIN_URL . '/assets/images/wechat.svg" class="stripe-wechat-icon stripe-icon" alt="Wechat Pay" />',
```

Comment:
I went back to previous commits to see when this issue might have been introduced. This `'wechat'` key was added 6 years ago in https://github.com/woocommerce/woocommerce-gateway-stripe/commit/5e4415c088ec22d8a0e2499e5393fdf0a9a8040f. That makes me think this icon was never shown on the classic checkout when we built the UPE method for it.